docs/sources/flow/setup/start-agent.md

```diff
@@ -150,26 +150,102 @@ If you downloaded the standalone binary, you must run the agent from a terminal
 To start Grafana Agent on Linux, macOS, or FreeBSD, run the following command in a terminal window:
 
 ```shell
-AGENT_MODE=flow BINARY_PATH run CONFIG_FILE
+AGENT_MODE=flow BINARY_PATH run CONFIG_PATH
 ```
 
 Replace the following:
 
-* `BINARY_PATH`: The path to the Grafana Agent binary file
-* `CONFIG_FILE`: The path to the Grafana Agent configuration file.
+* `BINARY_PATH`: The path and Grafana Agent binary filename.
```

Comment:
I'm not sure that the new language makes sense.

A path includes the filename already.

Reply:
Updated in https://github.com/grafana/agent/pull/5455